### PR TITLE
updates to linux install steps

### PR DESCRIPTION
**Pull request by mikeblome**
@gregvanl if these look good we can  go ahead and publish.

### CODE REVIEW

File: docs/cpp/config-wsl.md [hunk 1]
```diff
@@ -49,20 +49,24 @@ To successfully complete this tutorial, you must do the following steps:
...
 
    ```bash
-   cd $HOME
-   cd ../../usr/bin
-   ls g++ gdb
+   sudo apt-get install build-essential gdb
    ```
+1. Verify that the install succeeded by navigating to the /user/bin directory and locating g++ and gdb. If the filenames are not returned from the following command, try running the update and upgrade command again.
 
+```bash
```
Comment (gregvanl) [inline docs/cpp/config-wsl.md]: Can you indent this code block 3 spaces? Otherwise it won't be indented correctly under the step text.

Comment (gregvanl) [reply docs/cpp/config-wsl.md]: I can do it to get this out today.

Comment (mikeblome) [reply docs/cpp/config-wsl.md]: @gregvanl thanks I missed that. Done.

Comment (mikeblome) [reply docs/cpp/config-wsl.md]: ok never mind i see you took care of it. thanks!

**Comment by gregvanl** [comment]
Published

**Pull request merged**